rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1828,7 +1851,12 @@ protected void executeOpcode() {
             break;
             case OpCodes.OP_CALL:
             case OpCodes.OP_CALLCODE:
-            case OpCodes.OP_DELEGATECALL: doCALL();
+            case OpCodes.OP_DELEGATECALL:
+            case OpCodes.OP_STATICCALL:
+                if (!config.isRcs230()) {
```

Comment:
[BUG] This check should only apply to `OP_STATICCALL`, not all `CALL` opcodes.

Reply:
fixed